javascript/src/main/components/Schedule/AddToScheduleTable.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import BootstrapTable from 'react-bootstrap-table-next';
+import { Button } from "react-bootstrap";
+
+const AddToScheduleTable = ({ data }) => {
+
+  const renderAddToButton = (id) => {
+     return (
+       <Button data-testid={`add-to-button-${id}`}>Add To</Button>
+     )
+  }
+
+  const columns = [{
+    dataField: 'id',
+    text: 'id'
+  }, {
```

Comment:
I am slightly bothered by the fact that this line isn't consistent with the rest of this array lol
It's up to you if you would like to fix it

Reply:
fixed!